adhocracy-plus/templates/a4dashboard/base_dashboard_project.html

```diff
@@ -138,3 +138,8 @@
         </div>
     </div>
 {% endblock %}
+
+{% block extra_js %}
```

Comment:
Do we also have a dashboard_base? Or will we also get problems when we use that?

Reply:
Yes, we do have a dashboard_base. I just did it the way its done in meinBerlin, there its added to base_dashboard_project. But I actually dont think we get problems if I add it to dashboard base. So should I do that instead?

Reply:
As you like! You can just merge or move it to dashboard_base. Well, I think, I prefer the latter! :)